etc/initsystem/prepare-dirs.cmake

```diff
@@ -3,6 +3,11 @@
 # This script prepares directories and files needed for running Icinga2
 #
 
+# With sysvinit we need to pass the environment this way. Or do we?
```

Comment:
If this gets merged, the comment should be updated.

Reply:
Aye, the ramblings of my thought process are not really required here ^_^